cmd/kubectl-gadget/deploy.go

```diff
@@ -92,11 +94,34 @@ var (
 	eventBufferLength   uint64
 	daemonLogLevel      string
 	appArmorprofile     string
+	verifyImage         bool
+	publicKey           string
 	strLevels           []string
 )
 
 var supportedHooks = []string{"auto", "crio", "podinformer", "nri", "fanotify", "fanotify+ebpf"}
 
+var clusterImagePolicyKind = schema.GroupVersionKind{
+	Group:   "policy.sigstore.dev",
+	Version: "v1beta1",
+	Kind:    "ClusterImagePolicy",
+}
+
+var admissionControllerFormat = `
+apiVersion: policy.sigstore.dev/v1beta1
+kind: ClusterImagePolicy
+metadata:
+  name: %s-image-policy
+spec:
+  images:
+  - glob: "**"
```

Comment:
Why not set this? I think it'll broke deployments on any namespace with the annotation:

```bash
# after deploying Inspektor Gadget
$ kubectl create ns foo
...
$ kubectl label namespace foo policy.sigstore.dev/include=true
...
$ kubectl run -n foo -ti mypod --rm --image=ubuntu -- /bin/sh
Error from server (BadRequest): admission webhook "policy.sigstore.dev" denied the request: validation failed: failed policy: gadget-image-policy: spec.containers[0].image
index.docker.io/library/ubuntu@sha256:77906da86b60585ce12215807090eb327e7386c8fafb5402369e421f44eff17e signature key validation failed for authority authority-0 for index.docker.io/library/ubuntu@sha256:77906da86b60585ce12215807090eb327e7386c8fafb5402369e421f44eff17e: no matching signatures
```

I think we can set if to `image`.